presto-main/src/main/java/com/facebook/presto/operator/aggregation/fixed_histogram/FixedDoubleBreakdownHistogram.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.fixedhistogram;
```

Comment:
Remove the underscore in the directory path: `fixed_histogram`

Reply:
Hey, 

Sorry, I don't understand the comment. It says to remove the underscore, but there's no underscore there (I already removed it). If the comment is to add an underscore, I'm very happy to do that, but it seemed to me that other packages didn't (e.g., `arrayagg`).